lib/views/reporting/reporting.dart

```diff
@@ -49,138 +69,119 @@ class ReportingViewState extends State<ReportingView> {
         ReportingCardView(
-          question:
-              "Welk diersoort hoort bij de ${widget.interactionType.name.toLowerCase()}?",
+          question: "Diersoort rapportage:",
           step: 1,
           buttonText: "Volgende",
+          goToPreviousPage: _pageController.previousPage,
           onPressed: () {
             _pageController.nextPage(
-              duration: Durations.long1,
+              duration: const Duration(milliseconds: 300),
               curve: Curves.linear,
             );
           },
           onDataChanged: _updateInteractionData,
+          interactionType: widget.interactionType,
         ),
         ReportingCardView(
-          question:
-              "Welk ${_selectedAnimalSpecies != null ? _selectedAnimalSpecies!.substring(0, _selectedAnimalSpecies!.length - 2).toLowerCase() : "diersoort"} hoort bij de ${widget.interactionType.name.toLowerCase()}?",
+          question: "Dier rapportage:",
           step: 2,
           buttonText: "Volgende",
           animalSpecies: _selectedAnimalSpecies,
+          goToPreviousPage: _pageController.previousPage,
           onPressed: () {
             _pageController.nextPage(
-              duration: Durations.long1,
+              duration: const Duration(milliseconds: 300),
               curve: Curves.linear,
             );
           },
           onDataChanged: _updateInteractionData,
+          interactionType: widget.interactionType,
         ),
       ],
-      ReportingCardView(
-        question:
-            "Dit is de locatie van je ${widget.interactionType.name.toLowerCase()}.",
-        step: 3,
-        buttonText: "Volgende",
-        species: _selectedSpecies,
-        onPressed: () {
-          _pageController.nextPage(
-            duration: Durations.long1,
-            curve: Curves.linear,
-          );
-        },
-        onDataChanged: _updateInteractionData,
-      ),
       if (widget.interactionType.id == 2) ...[
         ReportingCardView(
           question: "Heb je nog opmerkingen?",
-          step: 4,
+          step: 3,
           buttonText: "Overslaan",
+          goToPreviousPage: _pageController.previousPage,
           onPressed: () {
             _pageController.nextPage(
-              duration: Durations.long1,
+              duration: const Duration(milliseconds: 300),
               curve: Curves.linear,
             );
           },
           location: _reportLocation,
+          animalSpecies: _selectedAnimalSpecies,
           species: _selectedSpecies,
           interactionType: widget.interactionType,
           onDataChanged: _updateInteractionData,
         ),
       ],
       ReportingCardView(
-        question: "Klopt je rapportage?",
-        step: 5,
+        question: "Overzicht rapportage:",
+        step: 4,
         buttonText: "Rapporteer",
+        goToPreviousPage: _pageController.previousPage,
         onPressed: () {
           _submitReport();
```

Comment:
Ik zou deze even awaiten

Reply:
Dit is een functie die je door geeft naar een andere widget, dus await is niet nodig.

Reply:
Ik bedoel hier de submitreport functie, misschien dat dat een beetje onduidelijk was maar ik neem aan dat die wel gewoon ge-await kan worden